op-acceptance-tests/tests/base/rpc_connectivity_test.go

```diff
@@ -33,28 +33,35 @@ func rpcConnectivityTestScenario() systest.SystemTestFunc {
 			t.Run(fmt.Sprintf("L2_Chain_%d", chainIdx), func(t systest.T) {
 				require.NotEmpty(t, l2Chain.Nodes(), "L2 chain has no nodes")
 
-				// Get the first node
-				execNode := l2Chain.Nodes()[0]
-
-				// Connect to the node's RPC
-				client, err := execNode.GethClient()
-				require.NoError(t, err, "failed to connect to L2 execution RPC")
-
-				// Check if we can get chain ID
-				chainID, err := client.ChainID(ctx)
-				require.NoError(t, err, "failed to get chain ID from L2 execution RPC")
-				require.Equal(t, chainIdx, chainID, "L2 chain ID is not correct")
-
-				// Check if we can get the latest block number
-				blockNumber, err := client.BlockNumber(ctx)
-				require.NoError(t, err, "failed to get block number from L2 execution RPC")
-				require.Greater(t, blockNumber, uint64(0), "L2 block number should be greater than 0")
-
-				logger.Info("L2 execution RPC connectivity test passed",
-					"chain", chainIdx,
-					"node", execNode.Name(),
-					"chain_id", chainID,
-					"block_number", blockNumber)
+				// Get the expected chain ID from the L2Chain
+				expectedChainID := l2Chain.ID()
+
+				// Test connectivity with each node in the L2 chain
+				for nodeIdx, execNode := range l2Chain.Nodes() {
+					nodeIdx := nodeIdx
```

Comment:
I don't think this is necessary unless you are spinning up go routines to run concurrent tests. Its a little confusing to copy `nodeIdx` but not `execNode` since both get used in the code block below. I suggest either copying both or neither for consistency

Reply:
Agreed; removed.